generators/typescript-axios/templates/api.mustache

```diff
@@ -0,0 +1,60 @@
+/* tslint:disable */
+/* eslint-disable */
+{{>licenseInfo}}
+
+{{^withSeparateModelsAndApi}}
+import type { Configuration } from './configuration';
+import type { AxiosPromise, AxiosInstance, RawAxiosRequestConfig } from 'axios';
+import globalAxios from 'axios';
+{{#withNodeImports}}
+// URLSearchParams not necessarily used
+// @ts-ignore
+import { URL, URLSearchParams } from 'url';
+{{#multipartFormData}}
+import FormData from 'form-data'
+{{/multipartFormData}}
+{{/withNodeImports}}
+// Some imports not used depending on template conditions
+// @ts-ignore
+import { DUMMY_BASE_URL, assertParamExists, setApiKeyToObject, setBasicAuthToObject, setBearerAuthToObject, setOAuthToObject, setSearchParams, serializeDataIfNeeded, toPathString, createRequestFunction } from './common';
+import type { RequestArgs } from './base';
+// @ts-ignore
+import { BASE_PATH, COLLECTION_FORMATS, BaseAPI, RequiredError, operationServerMap } from './base';
+
+{{! ONFIDO: Added lines of code - BEGIN }}
+import { FileTransfer } from './file-transfer';
+import { AxiosHeaders } from 'axios';
+
+globalAxios.interceptors.response.use(async (response) => {
```

Comment:
Is this necessary, or what's desired here? I thought the point of FileTransfer was to work as **input** arg for multipart requests, not to abstract responses.

For instance, [this call from tests](https://github.com/onfido/onfido-node/blob/master/test/test-helpers.ts#L91): I'd expect it to accept a FileTransfer object as well, which internally would work OOTB.